staff/sergiomons/js/challenges/count-word/scripts/count-word.js

```diff
@@ -0,0 +1,6 @@
+function countChart(str) {
```

Comment:
TIP 'use strict';

Reply:
WARN ```countChart```is incorrect. the correct function name is ```countWords````. rename it.

Reply:
WARN file name is incorrect. rename it to ```countWords.js``` instead.

Reply:
Ok, I,m rename it. Thanks!